Update README.md (#13)
danielfett/inetbox.py: README.md

There is also a systemd service to run the script at every startup. The details are explained below.

## Adding /home/pi/.local/bin to your PATH

At the end of the above-described installation, you might get the following warning:
```Installing collected packages: PyYAML, paho-mqtt, pyserial, miqro, bitstruct, inetbox-py
  WARNING: The scripts pyserial-miniterm and pyserial-ports are installed in '/home/pi/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
  WARNING: The script truma_service is installed in '/home/pi/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
```

> Then add /home/pi/.local/bin to your PATH (i.e. at the end of ~/.bashrc) by the following commands. The first one adds `export PATH="$HOME/.local/bin:$PATH"` to the logged in user's .bashrc, and the second command reloads the .bashrc settings without the need to relogin:

Reviewer: ```suggestion
Then add `/home/pi/.local/bin` to your PATH (e.g., at the end of `~/.bashrc`) using the following commands. The first one adds `export PATH="$HOME/.local/bin:$PATH"` to the logged in user's `.bashrc`, and the second command reloads the `.bashrc` settings without the need to relogin:
```
